client/app/cdn/dedicated/domain/dashboard/cdn-dedicated-domain-dashboard.html

```diff
@@ -28,21 +28,21 @@
                                              data-align="end">
                                 <oui-action-menu-item data-text="{{ :: 'cdn_dedicated_domain_flush' | translate}}"
                                                       data-aria-label="{{ :: 'cdn_dedicated_domain_general_domain_action_flush_title' | translate:{ domain: $ctrl.cdnDomain.domain } }}"
-                                                      data-href="{{ $ctrl.$state.href('.flush') }}">
+                                                      data-href="{{ $ctrl.$state.href('app.networks.cdn.dedicated.domain.flush') }}">
                                 </oui-action-menu-item>
                                 <oui-action-menu-item data-ng-if="$ctrl.cdnDomain.status === 'on'"
-                                                      data-text="{{ :: 'cdn_dedicated_domain_general_domain_action_bypass' | translate }}"
+                                                      data-text="{{ :: 'cdn_dedicated_domain_bypass' | translate }}"
                                                       data-aria-label="{{ :: 'cdn_dedicated_domain_general_domain_action_bypass_title' | translate:{ domain: $ctrl.cdnDomain.domain } }}"
-                                                      data-on-click="setAction('domain/update/cdn-dedicated-domain-domain', $ctrl.cdnDomain)">
+                                                      data-href="{{ $ctrl.$state.href('app.networks.cdn.dedicated.domain.update') }}">
```

Comment:
`sref` ?

Reply:
this is not implemented in ui-kit which wants to be free of ui-router dependency.